docs/options.rst

```diff
@@ -78,10 +78,20 @@ Native Authenticator is based on username and password only. But if you need ext
     c.Authenticator.ask_email_on_signup = True
 
 
+Use reCaptcha to prevent scripted SignUp attacks
+------------------------------------------------
+
+Since anybody can sign up, you may want to use the lightweight single-click "I am not a robot" checkbox provided by reCAPTCHA v2 to reduce your risks from scripting attacks. You will need to register with reCaptcha at https://www.google.com/recaptcha/admin/create (you will need a Google account to do so). You can learn more about reCAPTCHA at https://developers.google.com/recaptcha/intro and if you would like to simply test this functionality without creating an account, you can do so as explained at https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do (do **NOT** do this in production).
+
+    c.Authenticator.recaptcha_key = "your key"
+    c.Authenticator.recaptcha_secret = "your secret"
+
+
 Force acceptance of Terms of Service before SignUp
 --------------------------------------------------
 
 You may force users to click a checkbox agreeing to TOS before they can Sign Up. This might be legally binding is some jurisditions.
+>>>>>>> 010da1be5d8b39b195df36f05a49b11dd9edcb7d
```

Comment:
This still needs to go

Reply:
Doh! Really sorry about that, not sure how that slipped in! Anyway, I've just removed it and updated the PR.

Thanks for catching it!